src/rollup/utils/email-valid.js

```diff
@@ -0,0 +1,13 @@
+/** @module pathfora/utils/email-valid */
+
+/**
+ * Validate that the string is a properly formatted email
+ *
+ * @exports emailValid
+ * @params {string} email
+ * @returns {boolean} valid
+ */
+export default function emailValid (email) {
+  var regex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/g;
```

Comment:
this regex will fail to properly validate several valid emails:
```
firstname.lastname@example.com
firstname+lastname@example.com
email@123.123.123.123
“email”@example.com
email@example-one.com
email@example.name
email@example.co.jp
```

probably better to use: 

```
var regex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```